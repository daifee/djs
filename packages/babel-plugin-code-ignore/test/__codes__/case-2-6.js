

import error from './error'; /* @code-ignore-statement kuwo */



;// @code-ignore-statement kuwo
function fun() {
  console.log('error');

  console.log('success'); // @code-ignore-statement kugou
}

error();