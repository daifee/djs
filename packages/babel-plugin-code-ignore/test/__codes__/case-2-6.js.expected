"use strict";

/**/

//
function fun() {
  console.log('error');
  console.log('success'); // @code-ignore-statement kugou
}

error();